frontend/src/__tests__/cypress/cypress/tests/e2e/pipelines/pipelines.cy.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  createOpenShiftProject,
+  deleteOpenShiftProject,
+} from '~/__tests__/cypress/cypress/utils/oc_commands/project';
+import { createDataConnection } from '~/__tests__/cypress/cypress/utils/oc_commands/dataConnection';
+import { createDSPASecret, createDSPA } from '~/__tests__/cypress/cypress/utils/oc_commands/dspa';
+import { replacePlaceholdersInYaml } from '~/__tests__/cypress/cypress/utils/yaml_files';
+import { ADMIN_USER } from '~/__tests__/cypress/cypress/utils/e2eUsers';
+import { AWS_BUCKETS } from '~/__tests__/cypress/cypress/utils/s3Buckets';
+
+import { projectListPage, projectDetails } from '~/__tests__/cypress/cypress/pages/projects';
+import { pipelineImportModal } from '~/__tests__/cypress/cypress/pages/pipelines/pipelineImportModal';
+import { createRunPage } from '~/__tests__/cypress/cypress/pages/pipelines/createRunPage';
+import {
+  pipelineDetails,
+  pipelineRunDetails,
+} from '~/__tests__/cypress/cypress/pages/pipelines/topology';
+
+const projectName = 'test-pipelines-prj';
+const dspaSecretName = 'dashboard-dspa-secret';
+const testPipelineName = 'test-pipelines-pipeline';
+const testRunName = 'test-pipelines-run';
+
+describe('An admin user can import and run a pipeline', { testIsolation: false }, () => {
+  before(() => {
+    // Provision a Project
+    createOpenShiftProject(projectName);
+
+    // Create a pipeline compatible Data Connection
+    const dataConnectionReplacements = {
+      NAMESPACE: projectName,
+      AWS_ACCESS_KEY_ID: Buffer.from(AWS_BUCKETS.AWS_ACCESS_KEY_ID).toString('base64'),
+      AWS_DEFAULT_REGION: Buffer.from(AWS_BUCKETS.BUCKET_2.REGION).toString('base64'),
```

Comment:
@FedeAlonso Can you make "BUCKET_2" not hardcoded, but an env variable, like "CY_AWS_BUCKET" ?
This way the user (or CI) can set it according to their bucket value (e.g. `CY_AWS_BUCKET=BUCKET_2`

Reply:
@FedeAlonso can you try to set `CY_AWS_BUCKET=BUCKET_2` and then call:
`AWS_DEFAULT_REGION: Buffer.from(AWS_BUCKETS[CY_AWS_BUCKET].REGION).toString('base64')`